admin/partials/boldgrid-backup-admin-backup-button.php

```diff
@@ -35,7 +34,7 @@
  * If the page is 'boldgrid-backup', then you're in the modal. The buttons will generally be the same,
  * but the formatting will be slightly different, hence the conditional below.
  */
-if ( 'boldgrid-backup' === $page ) {
+if ( false !== strpos( $page, 'boldgrid-backup' ) ) {
```

Comment:
This was not intentional. However, it appears that what is happening is, that because of the way the 'view details' page is loaded, that code never gets called.